### DISCUSSION TITLE
Health template should be publishable

### DISCUSSION THREAD

**Discussion by carlosmintfan** [laravel/framework#52207] [Ideas]
The health template (for usually the /up route) should be publish- and customizable because
1. You may simply want to change it, use Bootstrap instead of Tailwind etc.
2. It makes requests to CDNs which may cause GDPR problems. You may wish to self-host these resources.

Therefore, there should be a vendor:publish option giving me the ability to customize the template for the Laravel health route.

**Comment by AhmedAlaa4611** [comment]
I think it should be updated to `Tailwind CSS v4`
